source/manage-apps/delete/delete-with-ui.txt

```diff
@@ -16,10 +16,10 @@ Prerequisites
 -------------
 To delete an app, you must be logged in as a 
 :atlas:`Project Owner </reference/user-roles/#project-roles>` of the 
-`{+atlas+} <https://cloud.mongodb.com/user/register?tck=docs_realm>`_ 
+`MongoDB Atlas <https://cloud.mongodb.com/user/register?tck=docs_realm>`_ 
 project where you want to delete the app.
 You can learn more about creating an |atlas| account in the
```

Comment:
what's up w `|atlas|`? 

that's some snooty funkiness looks like. can you replace it w/ whatever word is there?

and maybe while at it, see if there are any other similar usages in the docs? 
this regex should find any instances of "|something-no-spaces|": `\|\S+\|`

Reply:
Good catch. Found the couple other instances and corrected.